core/src/main/java/com/valtech/aem/saas/core/indexing/IndexUpdateJobConsumer.java

```diff
@@ -0,0 +1,49 @@
+package com.valtech.aem.saas.core.indexing;
+
+import com.valtech.aem.saas.api.indexing.IndexUpdateService;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.sling.event.jobs.Job;
+import org.apache.sling.event.jobs.consumer.JobConsumer;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+@Component(service = JobConsumer.class, property = {
+    JobConsumer.PROPERTY_TOPICS + "=" + IndexUpdateJobConsumer.INDEX_UPDATE
+})
```

Comment:
please add `configurationPolicy = ConfigurationPolicy.REQUIRE` and provide a default (empty) config